src/core/diff/strategies/new-unified/__tests__/edit-strategies.test.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="jest" />
```

Comment:
Since `@types/mocha` and `@types/jest` both define `expect`, `beforeEach`, `afterEach`, etc we need to indicate which version of these global functions are being used. I'm not sure what the "correct" fix for this is, but it's unfortunate that we have this conflict.

Mocha is only needed for integration tests, and it seems like that could live in a separate package (kind of like `webview-ui`).